app/templates/main/request-received.html

```diff
@@ -15,7 +15,7 @@ <h2 class="govuk-heading-m govuk-!-margin-top-7">
             We will email a link to <span class="govuk-!-font-weight-bold">{{ context["user_email"] }}</span>.
         </p>
         <p class="govuk-body">
-            This may take anything up to 5 minutes to come through to your inbox.</p>
+            This may take up to 5 minutes to come through to your inbox.</p>
```

Comment:
This may take up to 5 minutes to be delivered to your inbox.

Reply:
Done!